### PR TITLE
[MXI-2393] fix range selection bug

### PR DESCRIPTION
**Pull request by sophiaknowles**
- The main file that I changed is `ZoomPanSelection.js`. Also added a sample rangeBar file which is the type of graph we are using for the sleep picker. The rest of the file updates are because the `dist` folder is checked in (useful since we are using this package in the smart alarm repo from this github fork)
- Added a check `e.type === 'mouseup'` so that when you are done dragging the selection area, the selection event handler is run. This is so that we know the new `xAxis` values whenever the selection area is changed
- The rest of the additions are all only run when `w.globals.isTimelineBar` is `true` (graph type of rangeBar). 
  - In most of these cases, the `yAxis` values replace the `xAxis` values since that is where the timestamp values are held. There were other examples of this in this file, they just hadn't been added everywhere necessary. These changes make sure that the proper timestamp values are sent back to the selection event handler on selection
  - The last case is in the `selectionDrawn` handler. Here, because the `startX` and `endX` properties are different in timelineBar graphs, I'm getting the drawn selection box and setting `startX` property to the `x` value and the `endX` value to that plus the `width` of the rectangle
- TL;DR **Bugs fixed**! -> mouseup listener so the selection event handler is properly run. `isTimelineBar` checks and logic updates for when default selection values are passed in, when selection is being dragged, and the selection is done drawing

### CODE REVIEW

**Comment by laurentrivard** [comment]
This looks great! We talked about this offline, but what's still needed in order to open a pull request against apexcharts and eventually use their published npm package instead of this fork?

**Comment by sophiaknowles** [comment]
> This looks great! We talked about this offline, but what's still needed in order to open a pull request against apexcharts and eventually use their published npm package instead of this fork?

Mainly just a decent amount of testing with rangeBar and other graph types to make sure functionality is not changed

**Comment by laurentrivard** [comment]
> > This looks great! We talked about this offline, but what's still needed in order to open a pull request against apexcharts and eventually use their published npm package instead of this fork?
> 
> Mainly just a decent amount of testing with rangeBar and other graph types to make sure functionality is not changed

👍  Let's open another ticket for this so we can eventually come back to this

**Comment by sophiaknowles** [comment]
https://whoopinc.atlassian.net/browse/MXI-2581
